src/components/manage/Controlpanels/Users/UserGroupMembershipControlPanel.jsx

```diff
@@ -0,0 +1,158 @@
+/**
+ * User Control Panel [user group membership management]
+ * TODO Enrich with features of user control panel. Then replace user control panel.
+ */
+import React, { useEffect } from 'react';
+import { find, startCase, toNumber } from 'lodash';
+import { Portal } from 'react-portal';
+import { useHistory } from 'react-router';
+import { Link, useLocation } from 'react-router-dom';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { useDispatch, useSelector } from 'react-redux';
+import { Container, Segment } from 'semantic-ui-react';
+import { Helmet, messages } from '@plone/volto/helpers';
+import {
+  getControlpanel,
+  getSystemInformation,
+  listActions,
+} from '@plone/volto/actions';
+import { Icon, Toolbar, Unauthorized } from '@plone/volto/components';
+import { getParentUrl } from '@plone/volto/helpers';
+import UserGroupMembershipMatrix from '@plone/volto/components/manage/Controlpanels/Users/UserGroupMembershipMatrix';
+import backSVG from '@plone/volto/icons/back.svg';
+import settingsSVG from '@plone/volto/icons/settings.svg';
+
+const UserGroupMembershipPanel = () => {
+  const intl = useIntl();
+  const history = useHistory();
+  const pathname = useLocation().pathname;
+  const dispatch = useDispatch();
+
+  const many_users = useSelector(
+    (state) => state.controlpanels.controlpanel?.data?.many_users,
+  );
+  const many_groups = useSelector(
+    (state) => state.controlpanels.controlpanel?.data?.many_groups,
+  );
+  const systeminformation = useSelector(
+    (state) => state.controlpanels.systeminformation,
+  );
+  const i_can_use_group_membership_panel = systeminformation
+    ? systeminformation.plone_version[0] >= 6 &&
```

Comment:
Why are we requiring Plone 6 here? I have removed this check and the control panel works as expected in Plone 5.2

Reply:
Can't reproduce anymore. I thought it was a problem with acl_users.searchUsers(id=query) with query == None. But I stepped back to Plone 5.2.7 and the problem is not reproducable.
So restriction can be removed.